cli/src/declarations.ts

```diff
@@ -436,6 +436,16 @@ export interface CapacitorConfig {
      * @default true
      */
     handleApplicationNotifications?: boolean;
+
+    /**
+     * On iOS 16.4 and greater, enable debuggable web content for release builds.
+     *
+     * This is always `true` for development builds.
+     *
+     * @since 4.7.4
```

Comment:
```suggestion
     * @since 4.8.0
```
This is a feature, so the next version would be 4.8.0, not 4.7.4

Also the other changes requested on https://github.com/ionic-team/capacitor/pull/6495 should also be addressed here